### PR TITLE
Add file

### PR DESCRIPTION
**Pull request by dbisiakowski**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    morseAnimationDiv.style.display = 'none';
+    window.clearInterval(showMorseId);
+    disableInput(false);
+}
+
+// Disable/enable textarea property.
+const disableInput = (status) => {
+    textInput.disabled = status;
+    morseInput.disabled = status;
+}
```
Comment (pankominek) [inline main.js:178]: Add one empty line to the end of the file.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    characterNumber = 0;
+    stopAnimationBtn.style.display = 'none';
+    playAnimationBtn.style.display = 'block';
+    morseAnimationDiv.style.display = 'none';
+    window.clearInterval(showMorseId);
+    disableInput(false);
+}
+
+// Disable/enable textarea property.
+const disableInput = (status) => {
```
Comment (pankominek) [inline main.js:175]: When the arrow function has 1 argument, () can be omitted.

const disableInput = status => {

File: style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    width: 100vw;
+    padding: 0;
+    margin: 0;
+}
+
+.morse {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
```
Comment (pankominek) [inline style.css:13]: Remove this empty line.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    width: 150px;
+    align-self: center;
+    border: #000000 1px solid;
+}
+
+#error {
+    font-size: 1.3em;
+    align-self: center;
+    color: #a52a52;
+}
```
Comment (pankominek) [inline style.css:67]: Add one empty line to the end of the file.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+#show-morse {
+    background-color: #000000;
+    height: 150px;
+    width: 150px;
+    align-self: center;
+    border: #000000 1px solid;
+}
+
+#error {
```
Comment (pankominek) [inline style.css:63]: Don't use id's to style elements.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+  <meta charset="utf-8">
+  <title>Morse Code Translator</title>
+  <meta name="description" content="The HTML5 Herald">
+  <meta name="author" content="SitePoint">
+  <link rel="stylesheet" href="./style.css">
+</head>
+<header class="morse-header">Morse Code Translator</header>
+<main class="morse">
+    <div id="error"></div>
```
Comment (pankominek) [inline index.html:13]: Remove this` <div>`. 
Insert error using js.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    <div class="morse-translator">
+        <div class="control">
+            <h2>Text</h2>
```
Comment (pankominek) [inline index.html:16]: Instead of the `<header>` tag, there should be` <label>` above the textarea.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+            <textarea class="textarea" id="text-input"></textarea>
+        </div>
+        <div class="control">
+            <h2>Morse Code</h2>
+            <textarea class="textarea" id="morse-input"></textarea>
+        </div>
+    </div>
+    <div class="morse-player">
```
Comment (pankominek) [inline index.html:24]: I suggest using `<ul><li></li></ul>`.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

File: index.html [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        <button class="btn hide" id="stop-btn">Stop play Morse</button>
+    </div>
+    <div class="morse-player">
+        <button class="btn" id="show-btn">Show Morse</button>
+        <button class="btn hide" id="hide-btn">Stop play Morse</button>
+    </div>
+    <div id="show-morse" class="hide"></div>
+</main>
+
+<body>
```
Comment (pankominek) [inline index.html:35]: Open `<body>` after closing the `<head>`.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body#Example

File: main.js [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+        o.connect(context.destination);
+        o.start();
+
+        if (morseInputWithoutSpaces[characterNumber] === '.') {
+            o.stop(context.currentTime + 0.05);
+        }
+        else if (morseInputWithoutSpaces[characterNumber] === '-') {
+            o.stop(context.currentTime + 0.2);
+        }
+        else {
```
Comment (pankominek) [inline main.js:111]: } else {
```suggestion
        } else {
```

File: style.css [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.morse-header {
+    font-size: 3vh;
+    background-color: #b5651d;
+    color: #fdfff5;
+    text-align: center;
+}
+
+.morse-translator{
```
Comment (pankominek) [inline style.css:23]: Add a space after the selector.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:2]: Removed.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,38 @@
+<!doctype html>
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:1]: Done.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Morse Code Translator</title>
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:6]: I used after meta, but I think that's not matter where you use title.
F.x. Google use title between meta and link

File: index.html [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+        <button class="btn" id="show-btn">Show Morse</button>
+        <button class="btn hide" id="hide-btn">Stop play Morse</button>
+    </div>
+    <div id="show-morse" class="hide"></div>
+</main>
+
+<body>
+  <script src="./main.js"></script>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:38]: Added.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+        <button class="btn" id="play-btn">Play Morse</button>
+        <button class="btn hide" id="stop-btn">Stop play Morse</button>
+    </div>
+    <div class="morse-player">
+        <button class="btn" id="show-btn">Show Morse</button>
+        <button class="btn hide" id="hide-btn">Stop play Morse</button>
+    </div>
+    <div id="show-morse" class="hide"></div>
+</main>
+
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:34]: Removed.

File: index.html [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Morse Code Translator</title>
+  <meta name="description" content="The HTML5 Herald">
+  <meta name="author" content="SitePoint">
+  <link rel="stylesheet" href="./style.css">
+</head>
+<header class="morse-header">Morse Code Translator</header>
```
[the remark this answers is not in the document]
Comment (dbisiakowski) [reply index.html:11]: Done.

File: main.js (hunk 1 above)
Comment (dbisiakowski) [reply main.js]: Added.

File: main.js (hunk 2 above)
Comment (dbisiakowski) [reply main.js:167]: Done.

File: style.css (hunk 3 above)
Comment (dbisiakowski) [reply style.css:13]: Removed.

File: style.css (hunk 5 above)
Comment (dbisiakowski) [reply style.css:63]: I think styling id is correct.

File: index.html (hunk 6 above)
Comment (dbisiakowski) [reply index.html:13]: Could You explain why?

File: index.html (hunk 7 above)
Comment (dbisiakowski) [reply index.html:16]: This is header of this page so I think it's correct.

File: index.html (hunk 8 above)
Comment (dbisiakowski) [reply index.html:24]: I'm not understand. I don't have there any list to use ul

File: index.html (hunk 9 above)
Comment (dbisiakowski) [reply index.html:35]: my mistake. Fixed.

File: main.js (hunk 10 above)
Comment (dbisiakowski) [reply main.js]: Already fixed.

File: style.css (hunk 11 above)
Comment (dbisiakowski) [reply style.css:23]: Added.

File: index.html (hunk 7 above)
Comment (pankominek) [reply index.html]: In my opinion, this `<h2>` tag is a label for textarea.
I think the page header is in the <header> tag.

File: index.html (hunk 8 above)
Comment (pankominek) [reply index.html]:
```
 <div class="morse-player">
        <button class="btn" id="show-btn">Show Morse</button>
        <button class="btn hide" id="hide-btn">Stop play Morse</button>
    </div>
```
I suggest replacing this code snippet with an unordered list.
The div tag will become more semantic.

File: index.html (hunk 14 above)
[the remark this answers is not in the document]
Comment (pankominek) [reply index.html:5]: This is not a mistake, but I think there is no point in mixing meta tags with the <title> tag.

File: index.html (hunk 7 above)
Comment (dbisiakowski) [reply index.html]: I used section instead div.

File: index.html (hunk 8 above)
Comment (dbisiakowski) [reply index.html]: I used section instead of div.

File: index.html [hunk 18]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Morse Code Translator</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header class="morse-header"><h1>Morse Code Translator</h1></header>
```
Comment (pankominek) [inline index.html:9]: Move `<h1>` to new line.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+    <main class="morse">
+        <div id="error"></div>
+        <section class="morse-translator">
```
Comment (pankominek) [inline index.html:12]: Sections should always have headers.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+            </section>
+        </section>
+        <section class="morse-player">
+            <button class="btn" id="play-btn">Play Morse</button>
+            <button class="btn hide" id="stop-btn">Stop play Morse</button>
+        </section>
+        <section class="morse-player">
+            <button class="btn" id="show-btn">Show Morse</button>
+            <button class="btn hide" id="hide-btn">Stop play Morse</button>
+        </section>
```
Comment (pankominek) [inline index.html:29]: Sections should always have headers.

File: main.js [hunk 21]
```diff
@@ -0,0 +1,170 @@
...
+    4: '....-',
+    5: '.....',
+    6: '-....',
+    7: '--...',
+    8: '---..',
+    9: '----.',
+    0: '-----'
+};
+
+const textInput = document.getElementById('text-input');
```
Comment (IwoHerka) [inline main.js:41]: Again, don't put everything into global space. Make an object representing program's state and pass it around.

Comment (dbisiakowski) [reply main.js]: Object created, also all eventListeneder moved to the bottom.

**Pull request merged**